### ISSUE TITLE
FR: Option to write DllPlugin manifest as a single line JSON

### ISSUE THREAD

**Issue by iamdanfox** [webpack/webpack#4477]
**Do you want to request a *feature* or report a *bug*?**
Feature request

**What is the current behavior?**
Using the DllPlugin means that a large json file will be committed to the repo... since this is a pretty-printed JSON, dependency changes can result in thousands of lines in a diff. (This is because of a hard-coded [2 character indent in LibManifestPlugin][1])

```js
        new webpack.DllPlugin({
            path: path.join(__dirname, 'dll', '[name]-manifest.json'),
            name: '[name]_lib'
        }),
```

**If this is a feature request, what is motivation or use case for changing the behavior?**
Motivation: I'd like better signal-to-noise ratio when reviewing PRs involving dependency changes. (I already use yarn so I can see granular transitive changes)
Desired behaviour: Let's add a config option to write out the manifest file as a giant one-line JSON so that the total line count of diffs are reduced.  Example usage could be:

```js
        new webpack.DllPlugin({
            path: path.join(__dirname, 'dll', '[name]-manifest.json'),
            prettyPrintManifest: false,
            name: '[name]_lib'
        }),
```

**Please mention other relevant information such as the browser version, Node.js version, webpack version and Operating System.**

webpack 1.13.2

[1]: https://github.com/webpack/webpack/blob/f66f02497a9ce53786b4f1ac70f34e2ad48e06b4/lib/LibManifestPlugin.js#L42

**Comment by sokra** [comment]
Send a PR changing it to a single line. It doesn't have to be pretty printed. We don't need an option to pretty print it. Every developer should be able to pretty print it, i. e. with node.js.

**Comment by timse** [comment]
do you need this for webpack 1 or 2 @iamdanfox ?

**Comment by alfrcr** [comment]
Has anyone working on this?

**Comment by alfrcr** [comment]
Can we close this issue as the PR has been merged #4794? @sokra

**Comment by TheLarkInn** [comment]
Yup! Closing, thnk you!
